resources/plugin.yml

```diff
@@ -59,11 +59,16 @@ permissions:
             towny.claimed.*: true
             towny.admin.nation_zone: true
             towny.admin.spawn: true
+            towny.outlaw.tpbypass: true
```

Comment:
Any admin node should start with the towny.admin.{foo}

I would suggest towny.admin.outlaw.teleportbypass, which can be added to the PermissionNodes.java underneath the towny.admin.nation_zone permission node.

Reply:
Make the towny.admin.outlaw.teleportbypass node a child node of towny.admin.

Reply:
Done